_includes/events.ejs

```diff
@@ -0,0 +1,40 @@
+---
+layout: base-layout.ejs
+pagination:
+    data: collections.event
+    size : 6
+    alias : events
+---
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <link href="/assets/stylesheets/events.css" rel="stylesheet"/>
+    </head>
+    <body>
+        <div class = "banner">
+            <div class = "banner-content">
+                <h1 class = "banner-subtitle">Events</h1>
+                <h2 class = "banner-title">Something for everyone !</h2>
+                <p>
+                    "We have 9 technical events as well as 7 non-technical events."

```

Comment:
make these counts dynamic, use a front-matter boolean variable `technical` for all events, filter the array of events and display their counts dynamically. You can also segregate the technical and non-technical events in the listings.